jep/0000/README.adoc

```diff
@@ -0,0 +1,469 @@
+= JEP-0000: Serve the Update Center JSON files from R2 CloudFlare buckets
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 0000
+
+| Title
+| Serve the Update Center JSON files from R2 CloudFlare buckets
+
+| Sponsor
+| link:https://github.com/lemeurherve[Hervé Le Meur], link:https://github.com/dduportal[Damien Duportal] :bulb:
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Process
+
+| Created
+| 2023/11/29
+
+| BDFL-Delegate
+| TBD
+
+//
+//
+// Uncomment if there is an associated placeholder JIRA issue.
+| JIRA
+| https://issues.jenkins.io/browse/INFRA-3100 https://issues.jenkins-ci.org/browse/INFRA-3100[INFRA-3100] (archived, replaced by https://github.com/jenkins-infra/helpdesk/issues/2649[help desk issue #2649])
+//
+//
+// Uncomment if discussion will occur in forum other than jenkinsci-dev@ mailing list.
+| Discussions-To
+| https://github.com/jenkins-infra/helpdesk/issues/2649
+//
+//
+// Uncomment if this JEP depends on one or more other JEPs.
+//| Requires
+//| :bulb: JEP-NUMBER, JEP-NUMBER... :bulb:
+//
+//
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+== Abstract
+
+// [TIP]
+// ====
+// Give a short (200 word) description of the technical issue addressed.
+
+// * Use present tense - describe what the proposal "does" (as if it were already done), not what it will do.
+// * Do not go into technical details and instead put those in the Specification section.
+// * Do not talk about history or why this needs to be done. Instead, add the history to the Motivation section.
+// ====
+
+updates.jenkins.io serves the JSON of the update center used by Jenkins instances all around the world to know what plugins are available for installation and which is their last version.
+
+This corresponds to around 10Tio of data served every month, representing one of the largest Jenkins infrastructure cost centers.
+
+This document describes how we can serve this content from R2 CloudFlare buckets (S3-like object storage) instead of a VM on AWS to greatly reduce infrastructure costs and improve our quality of service.
```

Comment:
```suggestion
This document describes how we can serve this content from a Mirrorbit geographically redirected download to some R2 CloudFlare buckets (S3-like object storage) for exemple,  instead of a VM on AWS to greatly reduce infrastructure costs and improve our quality of service as it would be geo-localized.
```